src/GrpcProtos/capture.proto

```diff
@@ -517,6 +517,23 @@ message GpuDebugMarker {
   Color color = 5;
 }
 
+message PresentEvent {
+  uint32 pid = 1;
+  uint32 tid = 2;
+
+  // Unlike other events, we choose the (start_timestamp_ns, duration_ns) pair
+  // as some present "start" events don't have a corresponding "stop".
+  uint64 begin_timestamp_ns = 3;
+  uint64 duration_ns = 4;
```

Comment:
What does duration for these events mean? Why is this not a single point in time event?

Reply:
Perhaps, `duration_ns` could help identify "dropped" frames?

Reply:
D3d9 and Dxgi emit "PresentStart" and "PresentStop" ETW events. While it is not yet entirely clear what the "PresentStop" represent, the plan is for "duration_ns" to represent the time interval between the start/stop pair.